beacon_node/beacon_chain/src/canonical_head.rs

```diff
@@ -1062,7 +1062,8 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         self.store_migrator.process_finalization(
             new_finalized_state_root.into(),
             new_view.finalized_checkpoint,
-            self.head_tracker.clone(),
+            todo!(),
+            // self.head_tracker.clone(),
```

Comment:
- See https://github.com/sigp/lighthouse/issues/1785#issuecomment-1898483553 for ideas